Cargo.toml

```diff
@@ -17,6 +17,6 @@ anyhow = { version = "1.0" }
 async-std = { version = "1.12",  default-features = false, features = ["attributes"]}
 serde = { version = "1.0", default-features = false, features = ["derive"]}
 
-fluvio = { git = "https://github.com/infinyon/fluvio", tag = "v0.11.5"}
-fluvio-connector-common = { git = "https://github.com/infinyon/fluvio", tag = "v0.11.5", features = ["derive"]}
+fluvio = { git = "https://github.com/infinyon/fluvio", rev = "af1fcea0b50a8243de3f2b8e613646aa8b74fc92" }
```

Comment:
(nit) Does it make sense to add a comment here with the `git describe` of the hash, so we have an idea of the commit in some relation to the last fluvio release.

Reply:
For some reasons, it doesn't show me any tags, only entry with initial branch